test/Visualization_Tests/src/Table_Spec.enso

```diff
@@ -37,9 +37,10 @@ visualization_spec connection =
     Test.group "Table Visualization" <|
         Test.specify "should wrap internal errors" <|
             bad_table = Database_Table.Value Nothing Nothing Nothing Nothing
-            vis = Visualization.prepare_visualization bad_table 2
-            json = JS_Object.from_pairs [["error", "Method `set_limit` of Nothing could not be found."]]
-            vis . should_equal json.to_text
+            result = Panic.catch Any (Visualization.prepare_visualization bad_table 2) caught_panic->
+                caught_panic.payload
+            message = "Method `set_limit` of Nothing could not be found."
+            result.to_display_text . should_equal message
```

Comment:
Shouldn't this test be removed altogether?

We are no longer wrapping the errors.

If we want to keep the test, I suggest to rename it to `should forward internal errors`. Since that is what the test is testing right now, correct?

Reply:
Right, I will keep the test but change the name.